examples/spark2/pom.xml

```diff
@@ -38,32 +38,6 @@
       <groupId>org.apache.carbondata</groupId>
       <artifactId>carbondata-spark2</artifactId>
       <version>${project.version}</version>
-      <exclusions>
-        <exclusion>
-          <groupId>org.apache.spark</groupId>
```

Comment:
why need remove these exclusions ?  in spark2 module, need to exclude 2.10 dependency jars

Reply:
now spark-2.1 is the default profile. no need to exclusions spark-2.1 dependencies in spark2 module.

Reply:
I moved these exclusions to spark module.